v0/destinations/mailchimp/config.js

```diff
@@ -1,31 +1,43 @@
-const getEndpoint = (dataCenterId, audienceId) => {
-  const mailChimpApi = "api.mailchimp.com";
-  const listsUrl = `https://${dataCenterId}.${mailChimpApi}/3.0/lists`;
-  return `${listsUrl}/${audienceId}`;
-};
+const { getMappingConfig } = require("../../util");
 
-const destinationConfigKeys = {
-  apiKey: "apiKey",
-  audienceId: "audienceId",
-  dataCenterId: "datacenterId"
-};
+const MAX_BATCH_SIZE = 10;
 
-const subscriptionStatus = {
+const MAILCHIMP_IDENTIFY_EXCLUSION = [
+  "email",
+  "firstName",
+  "lastName",
+  "FirstName",
+  "LastName",
+  "firstname",
+  "lastname",
+  "addr1",
+  "city",
+  "state",
+  "zip",
+  "phone"
+];
+const SUBSCRIPTION_STATUS = {
   subscribed: "subscribed",
   pending: "pending"
 };
 
-const validStatuses = [
+const VALID_STATUSES = [
   "subscribed",
   "unsubscribed",
   "cleaned",
   "pending",
   "transactional"
 ];
 
+const CONFIG_CATEGORIES = {
+  IDENTIFY: { name: "mailchimpMergeFieldConfig" }
+};
+const MAPPING_CONFIG = getMappingConfig(CONFIG_CATEGORIES, __dirname);
+
 module.exports = {
-  getEndpoint,
-  destinationConfigKeys,
-  subscriptionStatus,
-  validStatuses
+  MAX_BATCH_SIZE,
+  MAILCHIMP_IDENTIFY_EXCLUSION,
+  SUBSCRIPTION_STATUS,
+  VALID_STATUSES,
+  mergeConfig: MAPPING_CONFIG[CONFIG_CATEGORIES.IDENTIFY.name]
```

Comment:
By convention as this is a constant shouldn't this be all caps

Reply:
done